packages/lesswrong/lib/collections/comments/views.js

```diff
@@ -156,7 +156,7 @@ Comments.addView("allRecentComments", function (terms) {
 
 Comments.addView("recentComments", function (terms) {
   return {
-    selector: { score:{$gt:0}, deletedPublic: false},
+    selector: { score:{$gt:0}, deletedPublic: false, shortform: terms.hideShortform ? false : viewFieldAllowAny },
```

Comment:
I think this would be better if it were symmetric, so you could filter both for no-shortform and only-shortform.